java/src/jmri/implementation/MatrixSignalMast.java

```diff
@@ -374,16 +374,16 @@ public void updateOutputs(char[] bits) {
             log.debug("Empty char[] received");
         } else {
             for (int i = 0; i < outputsToBeans.size(); i++) {
-                // log.debug("Setting bits[1] = {} for output #{}", bits[i], i);
+                log.debug("Setting bits[1] = {} for output #{}", bits[i], i);
                 Turnout t = getOutputBean(i + 1);
                 if (t != null) {
                     t.setBinaryOutput(true); // prevent feedback etc.
                 }
                 if (bits[i] == '1' && t != null && t.getCommandedState() != Turnout.CLOSED) {
                     // no need to set a state already set
-                    t.setCommandedState(Turnout.CLOSED);
+                    t.setCommandedStateAtInterval(Turnout.CLOSED); // delayed on specific connection by its turnoutManager
                 } else if (bits[i] == '0' && t != null && t.getCommandedState() != Turnout.THROWN) {
-                    t.setCommandedState(Turnout.THROWN);
+                    t.setCommandedStateAtInterval(Turnout.THROWN); // delayed on specific connection by its turnoutManager
```

Comment:
The delay here needs to be per signal mast.  Different masts on the same layout may need different delays.

Reply:
I am planning to add a per mast setting to MatrixMast to take care of mechanical mast chracteristics asap.
But that is does not rule out the need for a per connection (thus, per manager) interval setting, just like the Sensor Debounce Delay has a global as well as a bean setting.
JMRI cannot know to what decoder a specific output (Turnout) is connected. It only knows the connection (by the memo). So keeping an output timer per connection is the closest we can get for network speed characteristics, even more so for Routes which can more easily contain Turnouts connected to different connections.